plugins/node/opentelemetry-instrumentation-express/src/utils.ts

```diff
@@ -145,3 +145,27 @@ export const asErrorAndMessage = (
   error instanceof Error
     ? [error, error.message]
     : [String(error), String(error)];
+
+/**
+ * Extracts the layer path from the route arguments
+ *
+ * @param args - Arguments of the route
+ * @returns The layer path
+ */
+export const getLayerPath = (args: unknown[]): string | undefined => {
+  if (args[0] instanceof RegExp) {
+    return args[0].toString();
+  }
+
+  if (typeof args[0] === 'string') {
+    return args[0];
+  }
+
+  if (Array.isArray(args[0])) {
+    return args[0]
+      .map(arg => (typeof arg === 'string' || arg instanceof RegExp ? arg : ''))
+      .join(',');
+  }
+
+  return;
+};
```

Comment:
Why are we using `args: unknown[]` when we only ever use the first argument `args[0]`?

Reply:
Updated 👍